rules/naming/tests/Rector/Assign/RenameVariableToMatchGetMethodNameRector/Fixture/clousure.php.inc

```diff
@@ -0,0 +1,19 @@
+<?php
+
+$datetime = new DateTime();
+array_map(function ($element) use ($datetime) {
+    $a = $datetime->getTimestamp() + $element;
+    return $a;
+}, [1, 2, 3])
+
+?>
```

Comment:
Having this file outside any function/class method means it will be included in test case. It's included, because reflection/function_exists/PHPStan needs it to reflect types correctly.

I usually wrap it like this:

```php
<?php

class ClosureClass
{
    public function run()
	{
		$datetime = new DateTime();
		array_map(function ($element) use ($datetime) {
		    $a = $datetime->getTimestamp() + $element;
		    return $a;
		}, [1, 2, 3])
	}
}
```

It also allows to add runnable interface in the future, to compare results.

```diff
<?php

-class ClosureClass
+class ClosureClass implements RunnableInterface
{
    public function run()
	{
		$datetime = new DateTime();
-		array_map(function ($element) use ($datetime) {
+		return array_map(function ($element) use ($datetime) {
		    $a = $datetime->getTimestamp() + $element;
		    return $a;
		}, [1, 2, 3])
	}
}
```